packages/framework/presence/src/presenceDatastoreManager.ts

```diff
@@ -178,7 +178,7 @@ export class PresenceDatastoreManagerImpl implements PresenceDatastoreManager {
 			forceBroadcast: boolean,
 		): void => {
 			// Check for connectivity before sending updates.
-			if (this.runtime.clientId === undefined) {
+			if (!this.runtime.connected) {
```

Comment:
Should we check for both? Or is the runtime.connected check sufficient?

Reply:
Just connected is sufficient here. Once ever connected, clientId will be defined henceforth. (Which is how we got the bug.)